src/Console/ModelsCommand.php

```diff
@@ -1093,7 +1093,11 @@ protected function getFactoryMethods($model)
         }
 
         $modelName = get_class($model);
-        $factory = get_class($modelName::factory());
+        try {
+            $factory = get_class($modelName::factory());
+        } catch (\Error $error) {
+            return;
+        }
```

Comment:
I would suggest to not silently do this, but at least emit a warning: we do the same in other places, e.g. https://github.com/barryvdh/laravel-ide-helper/blob/c5c9b2b9517696e1d17ce64790be42b8bb76a4d2/src/Console/ModelsCommand.php#L623

(achtung: did not test the code…)
```suggestion
        } catch (\Throwable $t) {
            $this->warn(sprintf('Error resolving factory from model of %s: %s', $modelName, $, $t->getMessage()));

            return;
        }
```